db/structure.sql

```diff
@@ -21,6 +21,8 @@ CREATE TYPE public.work_access AS ENUM (
 
 SET default_tablespace = '';
 
+SET default_table_access_method = heap;
+
 --
```

Comment:
every time i run a migration on localhost, i get this added to my structure.sql file  - i'm running Postgres 14.3 locally